realm/src/main/java/io/realm/internal/LinkView.java

```diff
@@ -30,58 +33,52 @@ public LinkView(Context context, Table parent, long columnIndexInParent, long na
         this.nativeLinkViewPtr = nativeLinkViewPtr;
     }
 
-    protected static native void nativeClose(long nativeLinkViewPtr);
```

Comment:
We have discussed this a number of times now. @cmelchior finds it easier to read while @emanuelez and I have no strong opinion on the matter.